find-file-in-project.el

```diff
@@ -610,7 +616,9 @@ BSD/GNU Find use glob pattern."
       (setq tgt
             (if is-finding-directory (format "-iwholename \"*%s\"" keyword)
               (ffip--create-filename-pattern-for-gnufind keyword)))
-      (setq fmt "%s . \\( %s \\) -prune -o -type %s %s %s %s -print")))
+      (setq fmt (concat "%s "
+                        (format "%s" ffip-find-pre-path-options)
```

Comment:
why `(format "%s" ffip-find-pre-path-options)` instead of `ffip-find-pre-path-options`

Reply:
Whoops, I must be out of my mind. Have a check at the new commit.